### PR TITLE
Expose the buffer

### PR DESCRIPTION
**Pull request by marcoow**
This is sometimes needed, e.g. when working with arrays of records. Of course we could add a macro that supported arrays or change the existing macro so that it could deal with arrays as well as individual records but I'm not sure that would be worth the added cognitive load.

Also I think having access to the `Buffer` might be helpful in other scenarios as well and doesn't really expose a huge API that we'll have to maintain.

### CODE REVIEW

**Comment by Turbo87** [comment]
I don't have any experience with this project yet, so I'm probably the wrong person to review this. 😞

**Comment by marcoow** [comment]
@Turbo87: adding @pangratz as well - reviewing is a great way to getting familiar with code bases though ;)

File: README.md [hunk 1]
```diff
@@ -116,9 +116,32 @@ and
...
+
+const Validations = buildValidations({
+  name: validator('presence', true)
+});
+
+export default Ember.Controller.extend({
+  data: computed('model', function() {
+    let owner = Ember.getOwner(this);
+    return Buffer.extend(Validations).create(owner.ownerInjection(), {
+      content: this.get('model')
```
Comment (pangratz) [inline README.md]: I think it would be good to add a sample for when a custom `Buffer` might be helpful?

Comment (marcoow) [inline README.md]: That would necessarily be more complex though - I'm not sure it's worth it putting that right in the README? Might make more sense to add some guides or so later?

Comment (pangratz) [inline README.md]: Seems good

**Pull request merged**